litmus-portal/okteto-dev-env-setup/README.md

```diff
@@ -34,31 +34,25 @@ This directory contains setup guide to start developing Litmus Portal on Okteto
 - <h4>STEP-1:</h4> Export your GitHub username to an environment variable.
 
   ```bash
-  OKTETO_NAMESPACE="<GitHub username>"
+  GITHUB_USERNAME="<GitHub username>"
   ```
 
-- <h4>STEP-2:</h4> Obtain the Litmus Portal development manifest using the given template.
+- <h4>STEP-2:</h4> Update this readme file by replacing the `GITHUB_USERNAME` variable in STEP-5 using the given command.
 
   ```bash
-  sed s/%OKTETO_NAMESPACE%/$OKTETO_NAMESPACE/g litmus-portal-dev-manifest-template.yml > env/litmus-portal-dev-manifest.yml
+  envsubst < README.md > my-env/README.md
   ```
 
-- <h4>STEP-3:</h4> Update this readme file by replacing the `%GITHUB_USERNAME%` variable in STEP-6 using the given command.
-
-  ```bash
-  sed s/%GITHUB_USERNAME%/$OKTETO_NAMESPACE/g README.md > env/README.md
-  ```
-
-- <h4>STEP-4:</h4> Go to the root folder of this cloned repository i.e. `litmus` then add, commit and push these changes to your GitHub.
+- <h4>STEP-3:</h4> Go to the root folder of this cloned repository i.e. `litmus` then add, commit and push these changes to your GitHub.
```

Comment:
You don't need to do this to deploy your version of the portal on okteto. If you put the button with a link to cloud.okteto.com/deploy, without the `repository` query param, okteto will infer the  github repo and  the branch. (except for cases where the browser doesn't send the referrer, like brave). For those cases I think it's easier to just tell the reader to go cloud.okteto.com, click deploy, and paste the repo URL. What do you think?

Reply:
You can also deploy a pipeline from the cli directly:
```
okteto pipeline deploy --repository https://github.com/rberrelleza/limut --branch dev
```

Reply:
okay then that solves the problem for readme update, i will be removing the query parameter.. @rberrelleza, yeah CLI is also good, will the referrer thing work with google chrome? If yes we can go with this change or will have to add the CLI deploy step.. cc: @ksatchit

Reply:
yes, the referer works on chrome. So far, only brave blocks the referer as far as I know.

Reply:
we can then remove the query parameter, brave is not used by many for development purposes.